src/components/date-picker/flatpickr-adapter/flatpickr-adapter.tsx

```diff
@@ -132,6 +135,10 @@ export class DatePickerCalendar {
                 );
                 break;
         }
+
+        if (this.formatter) {
+            this.picker.formatDate = this.formatter;
```

Comment:
I don't know how to unresolve a conversation so I will post another comment 😵‍💫

I still think it might be more explicit to set the formatter with a setter from the Picker. For someone reading the code it might be easier if we leave some breadcrumbs don't you think?

Reply:
Sorry 😊 
What do you think about this? 0cad3575917589f6a4c5861c5b3bccafd1667de5

Reply:
like it! 🍭